### PR TITLE
Dirty bug fix solving issue when process id starts with a number

### PR DESCRIPTION
**Pull request by clausmichele**
Solves this issue: https://github.com/Open-EO/openeo-odc/issues/7

Maybe it's not the most elegant way of solving the problem, but I needed this quickly. If you find a better way of managing numeric process ids we can adapt the code.

### CODE REVIEW

File: src/openeo_odc/map_processes_odc.py [hunk 1]
```diff
@@ -106,12 +106,12 @@ def map_data(id, process, kwargs):
...
     params = process['arguments']
     if 'result_node' in kwargs:
-        params['data'] = kwargs['result_node']
+        params['data'] = '_' + kwargs['result_node']
         if process_name != 'apply':
             params['reducer'] = {}
     else:
-        params['data'] = convert_from_node_parameter(params['data'],
-                                                     kwargs['from_parameter'])
+        params['data'] = '_' + convert_from_node_parameter(params['data'],
```
Comment (sophieherrmann) [inline src/openeo_odc/map_processes_odc.py:113]: Please remove the spaces at the end of the line

**Review by sophieherrmann — COMMENTED** [review]
That is really useful especially in combination with the Webeditor - thanks! Generally looks good

File: src/openeo_odc/map_processes_odc.py (hunk 1 above)
Comment (sophieherrmann) [reply src/openeo_odc/map_processes_odc.py:113]: It's line 114, sry

**Comment by clausmichele** [comment]
@sophieherrmann could you please check the merge conflicts? It's because I created the PR before the latest additions you did

File: src/openeo_odc/map_processes_odc.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def map_load_collection(id, process):
                     params['y'] = (lowLat,highLat)
     if 'temporal_extent' in process['arguments']:
         def exclusive_date(date):
-            return str(np.datetime64(date) - np.timedelta64(1, 'D')).split(' ')[0] # Substracts one day
+            return str(np.datetime64(date) - np.timedelta64(1, 'D')) + 'Z' # Substracts one day
```
Comment (sophieherrmann) [inline src/openeo_odc/map_processes_odc.py:51]: Why did you add the `Z`? This looks quite weird e.g. `'time': ['2018-06-15', '2018-06-16Z']` because the start time is not followed by a `Z`. And I'm not completely sure if we need it..?

**Comment by clausmichele** [comment]
Issue has has been solved. When merging this and the PR #10 there will be merge conflicts depending on the order of merges. After the merge of this I can rebase the other branch onto master to solve the conflicts and then merge it.
